cli/pkg/digger/digger.go

```
@@ -203,13 +204,14 @@
 
 	if !allowedToPerformCommand {
 		msg := reportPolicyError(job.ProjectName, command, requestedBy, reporter)
-		log.Println(msg)
+		slog.Error(msg)
 		return nil, msg, errors.New(msg)
 	}
 
 	err = job.PopulateAwsCredentialsEnvVarsForJob()
 	if err != nil {
-		log.Fatalf("failed to fetch AWS keys, %v", err)
+		slog.Error("failed to fetch AWS keys, %v", err)
```

golangci-lint (1.64.8)

slog: slog.Error arg "err" should be a string or a slog.Attr (possible missing key or value)

(govet)